src/server/modules/user/interfaces/UrlManagementService.ts

```diff
@@ -11,6 +11,7 @@ export interface UrlManagementService {
     shortUrl: string,
     longUrl?: string,
     file?: GoUploadedFile,
+    tags?: string[],
```

Comment:
small nit, can we clarify if this is meant to be `tags (type TagType)` or `tagStrings (type string)`?